spec/isocalendar.html

```diff
@@ -151,112 +151,153 @@ <h1>Temporal.ISO8601Calendar.prototype.dateDifference ( _one_, _two_, _options_
     </emu-clause>
 
     <emu-clause id="sec-temporal.iso8601calendar.prototype.year">
-      <h1>Temporal.ISO8601Calendar.prototype.year ( _date_ )</h1>
+      <h1>Temporal.ISO8601Calendar.prototype.year ( _dateOrDateTime_ )</h1>
```

Comment:
It could also be a YearMonth, should we call it something like `_objectWithYear_`? (Same goes for below: era(), daysInMonth(), daysInYear(), monthsInYear(), and isLeapYear() are the same; the argument to month() could be a YearMonth or MonthDay; and the argument to day() could be a MonthDay)